src/recipients/RecipientPicker.tsx

```diff
@@ -1,6 +1,7 @@
 import { parsePhoneNumber } from '@celo/phone-utils'
 import { isValidAddress } from '@celo/utils/lib/address'
 import { NameResolution, ResolutionKind } from '@valora/resolve-kit'
+import { debounce } from 'lodash'
```

Comment:
I'm not sure what happened with these re-orderings.  I'm trying to transition from emacs to vscode and maybe this is some vscode thing I did by accident?

Reply:
This is because of this setting https://github.com/valora-inc/wallet/blob/696fe1df4d3233fe0d54d712d09525682ea2b408/.vscode/settings.json#L42
This has been annoying me for a while, we should either remove this or make this part of lint / prettier.

Reply:
oh, i'll take a pass at updating eslint and/or prettier.